bigtable/client/row_reader_test.cc

```diff
@@ -173,7 +176,7 @@ TEST_F(RowReaderTest, EmptyReaderHasNoRows) {
   bigtable::RowReader reader(
       client_, "", bigtable::RowSet(), bigtable::RowReader::NO_ROWS_LIMIT,
       bigtable::Filter::PassAllFilter(), std::move(retry_policy_),
-      std::move(backoff_policy_), metadata_update_policy_,
+      std::move(backoff_policy_), std::move(metadata_update_policy_),
```

Comment:
Moving the `metadata_update_policy_` here will make it unusable for any futures calls.  Is that what you want?